cmd/skupper/skupper.go

```diff
@@ -351,6 +351,7 @@ installation that can then be connected to other skupper installations`,
 	cmd.Flags().StringVar(&routerCreateOpts.Router.NodeSelector, "router-node-selector", "", "Node selector to control placement of router pods")
 	cmd.Flags().StringVar(&routerCreateOpts.Router.Affinity, "router-pod-affinity", "", "Pod affinity label matches to control placement of router pods")
 	cmd.Flags().StringVar(&routerCreateOpts.Router.AntiAffinity, "router-pod-antiaffinity", "", "Pod antiaffinity label matches to control placement of router pods")
+	cmd.Flags().StringVar(&routerCreateOpts.Router.IngressHost, "router-ingress-host", "", "Host through which node is accesible when using nodeport as ingress.")
```

Comment:
not sure what 'node' means here? site?

Reply:
Kubernetes node

Reply:
Typo here: `accesible`.

Reply:
@grs if you select ingress as `nodeport` but leave the `ingressHost` empty, the generated
token does not seem to work (edge-host and inter-router-host fields are empty). Should it
be a required parameter when using `--ingress nodeport`?

Reply:
Agreed. Both issues fixed.